src/krux/pages/encryption_ui.py

```diff
@@ -59,9 +102,22 @@ def encryption_key(self):
         if key in (ESC_KEY, MENU_CONTINUE):
             return None
 
+        if creating:
```

Comment:
I think this line has to go inside the `if key` and inside the `if creating`

Reply:
Can you elaborate? What `if key`? What's the current error/issue? How does this solve it?

Reply:
Got it, and refactored. Thank!